src/events.ts

```diff
@@ -48,6 +48,10 @@ export const EVENT_MAP = {
   'user.updated': true,
   'user.watching.start': true,
   'user.watching.stop': true,
+  // AI events
+  'ai_indicator.update': true,
```

Comment:
@kanat could we rename `ai_indicator` to just `ai`?

Reply:
A better name would be `ai_bot` IMO

Reply:
All naming related concerns were addressed in this Slack thread:
- https://getstream.slack.com/archives/CE5N802GP/p1732625926145299